_rules/audio-transcript-2eb176.md

```diff
@@ -24,12 +31,12 @@ acknowledgments:
 
 This rule applies to every [non-streaming](#non-streaming-media-element) `audio` element that is:
 
-- playing; or,
-- has a [play button][] that is [visible][] and [included in the accessibility tree][].
+- autoplaying; or,
```

Comment:
What does "autoplaying" means? [Audio or video element avoids automatically playing audio](https://www.w3.org/WAI/standards-guidelines/act/rules/80f0bf/proposed/) has a bunch of conditions for it.

Also, if the video plays but not automatically (e.g. after user interaction, but maybe that interaction hides the play button thus removing the second condition), would the rule become inapplicable?

Reply:
I updated the PR description with how we decided on this in a meeting.

Reply:
I have no problem with using autoplay in the rule, but it needs to be somewhat defined or clarified. In #2030 and the following discussion, we've seen that "autoplay" is not as easy as we could think.

Does that refers to the `autoplay` attribute or something else? How do we handle the fact that browsers do not always respect it (due to engagement index, …)?

I think that "autoplaying" in somewhat ambiguous and needs to be clarified.